src/index.js

```diff
@@ -42,23 +44,35 @@ const defaultDeclareConnect = (decl = {}, config = {}) => (
   _declareConnect(decl, { killProps: ['params', 'query', 'router'], ...config })
 );
 
+let _containerCounter = 0;
+const localPrefix = '__local';
+export function isLocalKey(key) {
+  return key.indexOf(localPrefix) === 0;
```

Comment:
minor: if it's a prefix it would be better to use `startsWith`

EDIT: nah, who cares!